PowerPointLabs/Test/UnitTest/SyncLab/SyncLabPositionTest.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Microsoft.Office.Interop.PowerPoint;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+using PowerPointLabs.SyncLab.ObjectFormats;
+
+namespace Test.UnitTest.SyncLab
+{
+    [TestClass]
+    public class SyncLabPositionTest : BaseSyncLabTest
```

Comment:
If it's a UI window instead of msg box, u will need to write controller to control it. For example, check this file, its usage and implementation https://github.com/PowerPointLabs/PowerPointLabs/blob/master/PowerPointLabs/TestInterface/IColorsLabController.cs